ci/debian-13/Dockerfile

```diff
@@ -1,32 +1,36 @@
-FROM debian:11
+FROM debian:trixie
```

Comment:
Do we want to call this `debian:trixie`? Are `debian:13`-tagged images available? I'm just looking at consistency with the previous debian Dockerfiles as well as the ubuntu ones.

Reply:
Ah, interesting. I've copied the `debian-12/Dockerfile` and that one had bookworm in it, but git now shows it as rename :-)

I left the `debian-12/Dockerfile` alone, but bumped zeekctl and builtin tests also to Debian 13.0 in the last push :crossed_fingers: